tests/TestGui/log_header.tcl

```diff
@@ -47,5 +46,13 @@ close $fp
 set found [regexp "^Copyright 20\\d\\d The Foedag team" $file_data]
 if { !$found } { puts "ERROR: foedag.log's header is commented which shouldn't have occured"; exit 1 }
 
+# Verify foedag.log has ANL: abbreviation in front of analyze messages
```

Comment:
Didn't we decide on having the format as shown below ?
`<log-level>: <task/module>: <message>`

Reply:
Regular Message() calls don't have a level unless you want STATUS: in front of every message?